buổi-10/main.js

```diff
@@ -0,0 +1,150 @@
+// bài 1
+const getMultiplication = number => {
+    if (number >= 1 && number <= 10) {
+        for(let i = 1; i <= 10; i++) {
+            console.log(number + ' * ' + i + ' = ' + (number * i))
```

Comment:
Có thể dùng string template cho đỡ rối mắt
```suggestion
            console.log(`${number} * ${i} = ${number * i}`)
```